### PR TITLE
Change add watched/ to-watch method

### PR DESCRIPTION
**Pull request by chao2048**
Change the way of adding watched/ to-watch movie. The program will show relevant result to the user, and the user can then pick the one they are looking for to add to the list.

### CODE REVIEW

**Review by choongzhanhong — COMMENTED** [review]
LGTM but some comments on underlying algorithm and one style comment

File: data/moviemate_data.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
 tt0111414|A Thin Life|2018|75|75,Comedy
+tt5440946|A Captain Unafraid|2016|96|"Adventure,Biography,Documentary"
+tt5715068|"19, The Tony Robinson Shooting: a Case of Deadly Bias"|2016|60|Documentary
```
[the remark this answers is not in the document]
Comment (chao2048) [reply data/moviemate_data.txt:3]: I think we should add it to .gitignore

File: src/main/java/MovieDatabase.java [hunk 2]
```diff
@@ -30,4 +32,25 @@ public MovieDatabase(ArrayList<String[]> movieStrings) throws IOException {
...
+    /**
+     * Find the movies that contain the movie name entered by the user and return them
+     *
+     * @param userInputMovieName The movie name entered by the user
+     * @return return an array list of movies with at most five movies, whose names containing the user input movie name
+     */
+    public static ArrayList<Movie> find(String userInputMovieName) {
+        ArrayList<Movie> relevantMovies = new ArrayList<Movie>();
+        Integer movieCount = 0;
+        for (Map.Entry<String, Movie> entry : movieDatabase.entrySet()) {
```
[the remark this answers is not in the document]
Comment (chao2048) [reply src/main/java/MovieDatabase.java:45]: Yes, the underlying algorithm is linear search. I think we can discuss about this during our meeting.

**Review by choongzhanhong — COMMENTED** [review]
LGTM

**Pull request merged**